tbx/settings/base.py

```diff
@@ -415,7 +415,7 @@
 PERMISSIONS_POLICY = {
     "accelerometer": [],
     "ambient-light-sensor": [],
-    "autoplay": ["self"],
+    "autoplay": ["self", "https://www.youtube-nocookie.com"],
```

Comment:
query: what does this setting do?

Reply:
The [default allowlist](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy/autoplay#default_policy) for `autoplay` is `self`.

This fixes an issue in Google Chrome, where, when you press the Play button, the video doesn't immediately play, forcing you to click the play button. I didn't encounter it in Firefox, I'm not sure about other browsers.

While this seems like we are setting videos to autoplay when the page loads, this is not the case. With the
`lite-youtube-embed` component, the video only loads when the user has clicked the play button.

Reply:
thanks for explaining.